.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: java
 install: mvn -pl "!doc" install -DskipTests=true -Dmaven.javadoc.skip=true -B -V
 script: 
-- java -cp environment/target/tygronenv-*-jar-with-dependencies.jar login.Login $TYGRONPWD $TYGRONMAIL
+- java -cp environment/target/tygronenv-*-jar-with-dependencies.jar login.Login $email $password
```

Comment:
yes mail should be before the password... I accidentaly did it wrong the first time